service-api/src/main/java/greencity/dto/courier/CreateCourierDto.java

```diff
@@ -12,8 +13,12 @@
 @ToString
 public class CreateCourierDto {
     @NotNull
+    @Pattern(regexp = "[A-Za-zА0-9'\\s]{1,30}",
+        message = "use English letters, no longer than 30 symbols")
```

Comment:
Same here
is the case when the name starts from a number or not a capital letter valid?

Reply:
should we add to the message that the name could contain numbers and whitespaces

Reply:
fixed